Fulfillment/APIs/In progress orders.md

```diff
@@ -0,0 +1,156 @@
+
+## Request
+
+### Endpoint
+
+`https://<host>/api/solr-query`
+
+Example: https://demo-oms.hotwax.io/api/solr-query
+
+### Header
+
+#### Body
+
+```
+{
+  "json": {
+    "params": {
+      "rows": "10",
+      "sort": "orderDate asc",
+      "group": true,
+      "group.field": "picklistBinId",
+      "group.limit": 1000,
+      "group.ngroups": true,
+      "q.op": "AND",
+      "start": 0
+    },
+    "query": "(*:*)",
+    "filter": "docType: OISGIR AND picklistItemStatusId: PICKITEM_PENDING AND -fulfillmentStatus: Rejected AND -shipmentMethodTypeId: STOREPICKUP AND facilityId: Store_1 AND productStoreId: STORE"
+  }
+}
+```
+
+| Parameter                | Description                                                                                                   | Required (Y/N) |
+|--------------------------|---------------------------------------------------------------------------------------------------------------|----------------|
+| `rows`                   | The number of groups.                                                                                        | No.            |
+| `group`                  | Filter to group orders.                                                                                      | Yes.           |
+| `group.field`            | The field to be grouped.                                                                                      | Yes.           |
+| `group.limit`            | The maximum number of items allowed in the group.                                                             | Yes.           |
+| `group.ngroups`          | The number of groups that have matched the query in the results. The default value is false.                   | No.            |
+| `defType`                | Selects the query parser to be used to process the query.                                                    | No.            |
+| `q.op`                   | Specifies the default operator for query expressions, overriding the default operator specified in the Schema. Possible values are "AND" or "OR".                                                                                       | No.            |
+| `qf`                     | The query fields.                                                                                             | No.            |
+| `docType`                | Reference index.                                                                                              | No.            |
+| `start`                  | Index page number.                                                                                            | No.            |
+| `orderTypeId`            | The ID of the order type in HotWax.                                                                           | Yes.           |
+| `picklistItemStatusID`   | The status of the order if picked or not. Default value: N.                                                   | No.            |
```

Comment:
```suggestion
| `picklistItemStatusID`   | The picking status of the item. Default value: N.                                                   | No.            |
```

Why would the default value for picklistItemStatusID be N? wouldn't it be some kind of ID?